Chessington.GameEngine/Helpers.cs

```diff
@@ -0,0 +1,57 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Chessington.GameEngine
+{
+    public class Helpers
+    {
+        public void AddMoves(
+            Board board, 
+            List<Direction> directions, 
+            Square currentSquare, 
+            int spacesAway, 
+            List<Square> moves, 
+            Player player
+            )
```

Comment:
Minor but I'd change the indentation on this to be in line with the opening curly bracket 🙂